encord/storage.py

```diff
@@ -771,17 +894,40 @@ def delete_storage_items(self, item_uuids: List[UUID], remove_unused_frames=True
     def refetch_data(self) -> None:
         """
         Refetch data for the folder.
+
+        Returns:
+            None
         """
         self._set_orm_folder(
             self._api_client.get(f"storage/folders/{self.uuid}", params=None, result_type=orm_storage.StorageFolder)
         )
 
     def _set_orm_folder(self, orm_folder: orm_storage.StorageFolder) -> None:
```

Comment:
Private method. Docstring optional. Not necessarily a bad move.

Reply:
I'll leave them in.